common/logging/log4rs-sample.yml

```diff
@@ -46,7 +83,31 @@ loggers:
 
   # Route log events sent to the "comms" logger to the "network" appender
   comms:
-    level: info
+    level: trace
+    appenders:
+      - network
+    additive: false
+  # Route log events sent to the "p2p" logger to the "network" appender
+  p2p:
```

Comment:
I wonder if p2p and comms shouldn't be in the same log? They are very closely related. @sdbondi ?

Reply:
Oh I see they go to the same appender